proteuslib/tools/parameter_sweep.py

```diff
@@ -329,6 +354,12 @@ def parameter_sweep(model, sweep_params, outputs, results_file=None, optimize_fu
         debugging_data_dir (optional) : Save results on a per-process basis for parallel debugging
                                         purposes. If None no `debugging` data will be saved.
 
+        interpolate_nan_outputs (optional) : When the parameter sweep has finished, interior values
+                                             of np.nan will be replaced with a value obtained via
+                                             a linear interpolation of their surrounding valid neighbors.
+                                             If true, a second output file with the extension "_clean"
+                                             will be saved alongside the raw (un-interpolated) values. 
+
```

Comment:
For another PR: we should make parameter_sweep a class, and use a Pyomo config object to manage these ever-growing options. That said, I would like to wait until we are feature-stable to do this.

Reply:
Agreed, the argument list is starting to feel a little cumbersome, so it's good to have this idea as part of the record.